go-controller/pkg/util/net.go

```diff
@@ -87,12 +87,14 @@ func GetOVSPortMACAddress(portName string) (net.HardwareAddr, error) {
 	return net.ParseMAC(macAddress)
 }
 
-// GetNodeWellKnownAddresses returns routerIP, Management Port IP and prefix len
-// for a given subnet
-func GetNodeWellKnownAddresses(subnet *net.IPNet) (*net.IPNet, *net.IPNet) {
+// GetNodeWellKnownAddresses returns the gateway address (the ".1" address),
+// the node Management Port address (the ".2" address), and the hybrid overlay
+// port address (the ".3" address) for a given subnet.
+func GetNodeWellKnownAddresses(subnet *net.IPNet) (*net.IPNet, *net.IPNet, *net.IPNet) {
```

Comment:
so of the 8 callers, only 1 actually wants all three IPs, and more than half only want one of the IPs. I think it would make more sense to have three separate functions.

Reply:
I dunno, with Go you can ignore what you don't want, so it seems more consistent to have a single function that returns all three which the caller can just ignore those they don't care about...

Reply:
But anyway, done.